tests/system/test_sqlalchemy_bigquery.py

```diff
@@ -774,3 +774,27 @@ def test_unnest(engine, bigquery_dataset):
         f" unnest(`{bigquery_dataset}.test_unnest_1`.`objects`) AS `foo_objects`"
     )
     assert sorted(r[0] for r in conn.execute(query)) == ["a", "b", "c", "x", "y"]
+
+
+@pytest.mark.skipif(
+    packaging.version.parse(sqlalchemy.__version__) < packaging.version.parse("1.4"),
+    reason="regexp_match support requires version 1.4",
+)
+def test_regexp_match(session, table):
+    results = session.query(table.c.string).where(
+        table.c.string.regexp_match(".*52 St &.*").all()
```

Comment:
We are getting error messages/failing tests for this assertion.

The result of a `regexp_match()` function call does not have an `all()` method ... I suspect we need the `.all()` method call to be segregated from the query via a set of closing parenthesis so that it is associated with the overall query. Right now, it is tied to the `regexp_match` and the closing parenthesis are on the following line. I think something closer to this without a closing parens on line 786 might do it.
```suggestion
def test_regexp_match(session, table):
    results = session.query(table.c.string).where(
        table.c.string.regexp_match(".*52 St &.*")).all()
```

**Error message**

```
Traceback (most recent call last):
  File "/tmpfs/src/github/python-bigquery-sqlalchemy/tests/system/test_sqlalchemy_bigquery.py", line 785, in test_regexp_match
    table.c.string.regexp_match(".*52 St &.*").all()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmpfs/src/github/python-bigquery-sqlalchemy/.nox/system-3-11/lib/python3.11/site-packages/sqlalchemy/sql/elements.py", line 857, in __getattr__
    util.raise_(
  File "/tmpfs/src/github/python-bigquery-sqlalchemy/.nox/system-3-11/lib/python3.11/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
AttributeError: Neither 'BinaryExpression' object nor 'Comparator' object has an attribute 'all'
```